src/test/java/com/artipie/api/RepositoryVerticleTest.java

```diff
@@ -34,14 +39,34 @@ class RepositoryVerticleTest {
      */
     private static final BlockingStorage ASTO = new BlockingStorage(new InMemoryStorage());
 
+    /**
+     * Service host.
+     */
+    private static final String HOST = "localhost";
+
     /**
      * Service port.
      */
     private static int port;
 
+    /**
+     * Maximum awaiting time duration of port availability.
+     * @checkstyle MagicNumberCheck (10 lines)
+     */
+    private static final long MAX_WAIT = Duration.ofMinutes(10).toMillis();
+
+    /**
+     * Sleep duration.
+     */
+    private static final long SLEEP_DURATION = Duration.ofMillis(100).toMillis();
+
+    static {
```

Comment:
Looks like over checking.
There is no needing to check constant value, it makes sense for function arguments.
I suggest removing these checking.

Reply:
removed.
@dgarus please review